acl_loader/main.py

```diff
@@ -596,16 +634,27 @@ def validate_rule_fields(self, rule_props):
             if ("ICMPV6_TYPE" in rule_props or "ICMPV6_CODE" in rule_props) and protocol != 58:
                 raise AclLoaderException("IP_PROTOCOL={} is not ICMPV6, but ICMPV6 fields were provided".format(protocol))
 
-    def convert_rule_to_db_schema(self, table_name, rule):
+    def convert_rule_to_db_schema(self, table_name, rule, rule_name, is_time_based):
```

Comment:
Better use an enum type instead. In future, we might introduce more ACL types, use enum can avoiding change the API:
- Standard
- TimeBased
- ...

Reply:
This is an optional comments. We can do the refactor after this PR for sure.

Reply:
Noted.